backend/btrixcloud/crawls.py

```diff
@@ -368,6 +375,27 @@ async def add_new_crawl(self, crawl_id: str, crawlconfig):
             # print(f"Crawl Already Added: {crawl.id} - {crawl.state}")
             return False
 
+    async def update_crawl(self, crawl_id: str, org: Organization, update: UpdateCrawl):
+        """Update existing crawl (tags only for now)"""
+        query = update.dict(
```

Comment:
nice - that makes this extensible if we want to add additional properties to update later.